parts/kubernetesmasteraddons-kube-dns-deployment.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
   name: kube-dns
   namespace: kube-system
 spec:
-  clusterIP: <kubeDNSServiceIP>
```

Comment:
Are you sure this is the right change? I remember there was work done to parametrize: `kubeDNSServiceIP`. Why remove it?

Reply:
Mistake. Must of been part igg a merge mistake. I'll revert it